Sources/NIO/NIOThreadPool.swift

```diff
@@ -175,7 +175,8 @@ public final class NIOThreadPool {
 
         for id in 0..<self.numberOfThreads {
             group.enter()
-            NIOThread.spawnAndRun(name: "NIOThreadPool thread #\(id)", detachThread: false) { thread in
+            // We should keep thread names under 16 characters because Linux doesn't allow more.
+            NIOThread.spawnAndRun(name: "TP-#\(id)", detachThread: false) { thread in
```

Comment:
I know I missed the boat on this one, but there may be some value in having NIO in the name: `"NIOTP-#\(id)"` perhaps?

Reply:
@glbrntt this time I deliberately left NIO out because we got so many bug reports about crashes if there's NIO in it :D . People thing that all crashes on NIO* threads are NIO's fault :|

Reply:
Oh, very sensible then! 😅